Cargo.toml

```diff
@@ -26,7 +26,7 @@ fs_extra = "1.2.0"
 futures-executor = "0.3.15"
 futures-util = "0.3.15"
 hex = "0.4.3"
-omnect-crypto = { git = "ssh://git@github.com/omnect/omnect-crypto.git", tag = "0.3.0" }
+omnect-crypto = { git = "https://github.com/omnect/omnect-crypto.git", tag = "0.3.0" }
```

Comment:
you should probably handle that first ... e.g. the current version of omnect-crypto is 0.4.0 and if you have to change something there, perhaps 0.4.1

Reply:
current version is 0.3.0 and there are no changes expected that are important for this repo

Reply:
done